src/plugins/crypto/README.md

```diff
@@ -109,62 +110,62 @@ Set the CMake variables `OPENSSL_INCLUDE_DIR` and `OPENSSL_LIBRARIES` to your de
 
 The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
-The key derivation is still WIP.
+The GPG binary is required for handling the cryptographic keys.
 
-### Planned Features ###
+At the moment the plugins will only run on UNIX/Linux-like systems, that implement `fork ()`.
 
-- Encryption of values
-- Decryption of values
-- Key derivation by metadata (password provided within a meta-key)
-- Key derivation by using a specified key-file (like the SSH client does)
-- Key derivation by utilizing the pgp-agent
+## Examples ##
+
+To mount a backend with the gcrypt plugin variant that uses the GPG key 9CCC3B514E196C6308CCD230666260C14A525406, use:
 
-The encryption and decryption of values is a straight forward process, once the key and IV are supplied.
-The key-derivation process in a library is a bit tricky.
+	kdb mount test.ecf user/t crypto_gcrypt "crypto/key=9CCC3B514E196C6308CCD230666260C14A525406"
 
-The crypto plugin itself can hold configuration data about the order of the applied key derivation functions.
-For example, if a password is provided by a meta-key, it will be used, otherwise we look for a key file.
-If no key file has been configured, we try to trigger the PGP agent, etc.
+Now you can specify a key `user/t/a` and protect its content by using:
 
-The following example configuration illustrates this concept:
+	kdb set user/t/a
+	kdb setmeta user/t/a crypto/encrypt 1
+	kdb set user/t/a "secret"
```

Comment:
What did you specify at `/spec`? I get the following error:

	root@258e61a6f05c:~/libelektra/build# bin/kdb mount test.ecf /t crypto_gcrypt crypto/key=7f230e8d
	root@258e61a6f05c:~/libelektra/build# bin/kdb global-mount spec
	The command global-mount terminated unsuccessfully with the info:
	It is not allowed to insert the same plugin (spec#spec) again!
	Try to add other plugins or other refnames (part after #) instead.